### DISCUSSION TITLE
Web3Wallet SDK launch

### DISCUSSION THREAD

**Discussion by chadyj** [WalletConnect/walletconnect-monorepo#1886] [Announcements]
Last year, WalletConnect achieved several milestones, among them the final release of the WalletConnect Sign and WalletConnect Auth APIs.

Along the way, we realized that there was a need to simplify the integration process for wallet developers, who already have so much on their plate. We also observed the reality that certain apps, such as those offering basic NFT profile picture features, will only seek to authenticate wallets, instead of opting for full transaction capabilities. With the growing popularity of these features, we need to help future-proof wallets so as to ensure compatibility, at all times, with any WalletConnect-enabled app.

That’s why we’ve taken the decision to bundle both the WalletConnect Sign and WalletConnect Auth APIs into a single package — the Web3Wallet SDK. It streamlines the integration process, making it easier for wallet developers to include the authentication and transaction signing features necessary for their users to connect and interact with all sorts of apps — now and in the future.

If you’re new to WalletConnect, you can get started immediately with the Web3Wallet SDK by heading over to our [docs](https://docs.walletconnect.com/2.0/web3wallet/about). If you’ve already integrated WalletConnect Sign and WalletConnect Auth, you’re all set and there is no action to take!

If you’re a wallet that has already integrated either WalletConnect Sign or WalletConnect Auth, you will absolutely be able to continue with your existing set-up. Nonetheless, we strongly encourage you to upgrade to the Web3Wallet SDK to future-proof against emerging use cases.

To demonstrate the simplicity of this new change, we will cover three basic actions (initialization, pairing, and responding to events) across two scenarios: for wallet developers who are currently using WalletConnect Sign and for wallet developers who are currently using WalletConnect Auth.

To learn more:

- [Read the blog post](https://medium.com/walletconnect/simplifying-integration-for-wallet-developers-with-the-new-web3wallet-sdk-8706b69e149c)
- [Check the docs](https://docs.walletconnect.com/2.0/web3wallet/about)

**Comment by hsvshfa** [comment]
https://github.com/WalletConnect/walletconnect-monorepo/discussions/1886#discussion-4783726

**Comment by ranjbarali33** [comment]
Airdrop

**Comment by memichelleva** [comment]
awesome )
